infra/binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/select/projection/engine/ProjectionsContextEngine.java

```diff
@@ -115,32 +114,38 @@ private boolean containsProjection(final OrderByItemSegment orderByItem, final C
     }
     
     private boolean isSameColumn(final Projection projection, final ColumnSegment columnSegment) {
-        Collection<ColumnProjection> columns = getColumnProjections(projection);
+        Collection<Projection> columns = expandProjection(projection);
```

Comment:
Do you think rename `expandProjection` to `getExpandedProjections` will be better?